packages/ui-settings/src/index.ts

```diff
@@ -2,10 +2,11 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import { Settings, settings } from './Settings';
+import { Endpoint, EndpointType, JsonRpcEndpoint, SubstrateConnectEndpoint } from './types';
 
 export { ENDPOINT_DEFAULT, ICON_DEFAULT, ICON_DEFAULT_HOST, LANGUAGE_DEFAULT, LOCKING_DEFAULT, PREFIX_DEFAULT, UIMODE_DEFAULT, UITHEME_DEFAULT } from './defaults';
 export { packageInfo } from './packageInfo';
 
-export { settings, Settings };
+export { settings, Settings, Endpoint, JsonRpcEndpoint, SubstrateConnectEndpoint, EndpointType };
```

Comment:
Remove the type exports here. (We always only have the types in `/types.ts`)

Reply:
> Remove the type exports here. (We always only have the types in `/types.ts`)

On next PR these types are needed. How can I import them there?